d-acts.owl

```diff
@@ -12,8 +12,8 @@
      xmlns:terms="http://purl.org/dc/terms/"
      xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#"
      xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#">
-    <owl:Ontology rdf:about="http://purl.obolibrary.org/obo/iao/d-acts.owl">
-        <owl:versionIRI rdf:resource="http://purl.obolibrary.org/obo/iao/d-acts/release/2021-01-08/d-acts.owl"/>
+    <owl:Ontology rdf:about="http://purl.obolibrary.org/obo/iao/d-acts/d-acts.owl">
```

Comment:
The change here doesn't resolve.  <http://purl.obolibrary.org/obo/iao/d-acts.owl> is currently working, while <http://purl.obolibrary.org/obo/iao/d-acts/d-acts.owl> does not.